proto/message_contents/invitation.proto

```diff
@@ -11,6 +11,73 @@ import "message_contents/public_key.proto";
 option go_package = "github.com/xmtp/proto/v3/go/message_contents";
 option java_package = "org.xmtp.proto.message.contents";
 
+// Unsealed invitation V2
+message InvitationV2 {
+    // Supported encryption schemes
+    // AES256-GCM-HKDF-SHA256
```

Comment:
Is the intention to use this Invitation for Invites sent to participants? Or only the invites sent to yourself?